src/Site_Command.php

```diff
@@ -115,6 +115,34 @@ private function empty_taxonomies() {
 		}
 	}
 
+	/**
+	 * Delete all links, link_category terms, and related cache.
+	 */
+	private function empty_links() {
+		global $wpdb;
+
+		// Remove links and related cached data.
+		$links_query = "SELECT link_id FROM $wpdb->links";
```

Comment:
Convention to always use curly braces on text expansion:
```suggestion
		$links_query = "SELECT link_id FROM {$wpdb->links}";
```